core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -436,7 +436,7 @@ private static MountInfo getRootMountInfo(MasterUfsManager ufsManager) {
       Map<String, String> rootUfsConf =
           ServerConfiguration.getNestedProperties(PropertyKey.MASTER_MOUNT_TABLE_ROOT_OPTION);
       MountPOptions mountOptions = MountContext
-          .defaults(MountPOptions.newBuilder().setShared(shared).putAllProperties(rootUfsConf))
+          .mergeFrom(MountPOptions.newBuilder().setShared(shared).putAllProperties(rootUfsConf))
```

Comment:
(Applies in general)
Should the `mergeFrom` be the other way around? This way will override the options from the mount context which should be respected?

Reply:
I'm not sure. Currently this will respect the previous implementation's behavior.

Reply:
Ah I see, the implemented `mergeFrom` is the opposite of the Protobuf `mergeFrom`. This is slightly confusing, could you leave a TODO to change the method name or have some other way of disambiguating this?

Reply:
It's actually similar behaviour to protobuf, but I agree that it doesn't communicate on with what it's merging from. If we decide to change the name, all Context.mergeFrom() should be being used from master side, so it should be an easy refactor.